src/qute/commands/registerCommands.ts

```diff
@@ -17,6 +19,11 @@ export function registerVSCodeQuteCommands(context: ExtensionContext) {
   registerJavaDefinitionCommand(context);
   registerConfigurationUpdateCommand(context);
   registerQuteValidationToggleCommand(context);
+  context.subscriptions.push(
+    workspace.onDidOpenTextDocument((document) => {
+      updateQuteLanguageId(context, document, true);
```

Comment:
I think it would be worth it to surround the update with :

```
if (!(document.uri.scheme === 'git')) {
...
}
```

like we do at https://github.com/redhat-developer/vscode-quarkus/blob/master/src/extension.ts#L59 .